src/_actions/modelNumbers.actions.js

```diff
@@ -28,6 +29,41 @@ export const loadModelNumbers = () => {
  *
  * @return {object} type and payload
  */
-export const loadModelNumbersSuccess = (modelNumbers) => {
-  return { type: LOAD_ASSET_MODEL_NUMBERS, payload: modelNumbers };
-}
+export const loadModelNumbersSuccess = modelNumbers => (
+  { type: LOAD_ASSET_MODEL_NUMBERS, payload: modelNumbers }
+);
+
+/**
+ * create new asset model number thunk
+ * @param {object} newModel details of new model to be created
+ *
+*/
```

Comment:
This is off...move forward by 1 space. Check the indentation of line 31 for a demonstration